lib/jsanalyze.js

```diff
@@ -26,6 +26,23 @@ const path = require('path');
 const SOURCE_MAPPING_URL_REGEXP = /\/\/#[ \t]+sourceMappingURL=([^\s'"`]+?)[ \t]*$/mg;
 const __ = appc.i18n(__dirname).__;
 
+function sortObject (o) {
```

Comment:
Please remove the space between the function and parens.

Reply:
:+1: fixed